packages/ui-extensions/src/surfaces/customer-account/targets.ts

```diff
@@ -70,6 +70,11 @@ export interface CustomerAccountExtensionTargets {
       FullPageApi,
     AllComponents
   >;
+  'customer-account.page.render': RenderExtension<
+    Omit<StandardApi<'customer-account.page.render'>, 'navigation'> &
+      FullPageApi,
+    AllComponents
+  >;
```

Comment:
Which extension target is this? A dynamic one that renders on every page?

Reply:
iirc that's replacing the full page extension target name

Reply:
yep, and @anahita-m  and i will be working on that part